labseu2-workout-tracker/server
# Authentication

##  What does this PR do
This PR allows users to register and login.

##  Description of task to be completed


##  How should this manually be tested
Use Postman to test endpoints: 
Signup: localhost:5000/api/auth/signup
body: {
	"username": "91111111",
	"password": "12345678" ,
	"email": "foobar2@gmail.com"
}

expect to receive back a message and newUser object

Login: localhost:5000/api/auth/login
body: {
	"password": "12345678" ,
	"email": "foobar2@gmail.com"
}

expect to receive back a token and user Id


##  What are the relevant Trello board stories
https://trello.com/c/kdGGnIJG/33-as-a-user-i-want-to-be-able-to-register-and-login-on-workout-tracker-app

##  Screenshots (If appropriate)


##  Questions (If any):
